lib/alchemy/admin/locale.rb

```diff
@@ -46,15 +46,16 @@ def available_locale
       def locale_from_user
         return if !current_alchemy_user
         if user_has_preferred_language?
-          current_alchemy_user.language
+          current_alchemy_user.language.to_sym
```

Comment:
Yes, you're right. 

I see that we are not consistent on the type here. We should always have symbols as the locale, since `I18n.locale` also ever returns symbols.

I would prefer to call `to_sym` on [`available_locale`](https://github.com/IngoAlbers/alchemy_cms/blob/141b6edc9f75e0cc6572aaa67e485e49442cc754/lib/alchemy/admin/locale.rb#L41) then:

```ruby
def available_locale
  locales = [params[:admin_locale], locale_from_user, locale_from_browser].compact.map(&:to_sym)
  locales.detect { |locale| ::I18n.available_locales.include?(locale) }
end
```

Then we are sure that all values are symbols